config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -285,17 +284,40 @@ config MBEDTLS_ENABLE_HEAP
 
 config MBEDTLS_HEAP_SIZE
     default 8192
+    
+# Enable PSA Crypto dependencies for Matter
 
-config NRF_SECURITY_ADVANCED
+config CHIP_CRYPTO_PSA
+    default n
+    select EXPERIMENTAL
```

Comment:
Sorry for bothering about this one :) But I think we cannot do it like this. Setting it here means that customer that doesn't use our defaults in the application (which is quite possible, because we source them only in our samples) will not get this warning. Can't we just add some dummy Kconfig without prompt, like:

```
config CHIP_CRYPTO_PSA_NRF_EXPERIMENTAL
     select EXPERIMENTAL
     depends on CHIP_CRYPTO_PSA
```
?

Reply:
Actually, technically, we can, and we overwrite CHIP_CRYPTO_PSA which is defined in Zephyr's config space. The other thing is that is clear and OK :)

Reply:
Eventually, I can remove it at all, and base on the OPENTHREAD_CRYPT_PSA kconfig, that is also experimental :)

Reply:
> Actually, technically, we can, and we overwrite CHIP_CRYPTO_PSA which is defined in Zephyr's config space. The other thing is that is clear and OK :)

Technically we can do everything, but the point is that it should print a warning to customer and it will not if defaults will not be loaded. I like the Damian's idea. We can override it in nrf space.